src/ui/public/visualize/visualization.js

```diff
@@ -126,6 +126,7 @@ uiModules
       });
 
       $scope.$on('$destroy', () => {
+        resizeChecker.off('resize');
```

Comment:
resize checker will only attach this listener `if (!$scope.vis.visualizeScope)` so you should probably check here as wel.